src/obstacle_avoidance/src/locate_hole.cpp

```diff
@@ -0,0 +1,33 @@
+#include <GoThroughHole.h>
+
+geometry_msgs::Point32 findHole (sensor_msgs::PointCloud scanData) {
```

Comment:
Both of these functions are a really good first start! I like where your current algorithm is heading and I think it shows a lot of promise.